service/order.rest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/rest-api-2.1.xsd"
+          name="order" displayName="Order REST API" version="${moqui_version}">
+    <resource name="orders" displayName="For order details" require-authentication="anonymous-all">
+        <method type="post">
+            <service name="tutorial.order.OrderServices.create#OrderHeader"/>
```

Comment:
Can we rename the service to create#OrderHeader to create#Order

Reply:
Sure,Ma'am